internal/serve/httperror/errors.go

```diff
@@ -60,9 +61,9 @@ func Unauthorized(message string, extras map[string]interface{}) *ErrorResponse
 	}
 }
 
-func InternalServerError(ctx context.Context, message string, err error, extras map[string]interface{}) *ErrorResponse {
-	// TODO: track error in Sentry
+func InternalServerError(ctx context.Context, message string, err error, extras map[string]interface{}, appTracker apptracker.AppTracker) *ErrorResponse {
 	log.Ctx(ctx).Error(err)
+	appTracker.CaptureException(err)
```

Comment:
Here we should check if the AppTracker is nil.

Reply:
Done